components/become-a-host/WhoAreFiscalHostsSection.js

```diff
@@ -119,9 +119,11 @@ const WhoAreFiscalHosts = () => {
           width={['222.74px', null, null, null, '330px']}
           height={['142.48px']}
         >
-          <Illustration
+          <NextIllustration
             src="/static/images/become-a-host/whoAreFiscalHost-illustration.png"
             alt="Who are Fiscal Hosts illustration"
+            width={330}
```

Comment:
There is a change on this one

![Peek 2021-05-07 09-04](https://user-images.githubusercontent.com/1556356/117410813-326a5180-af13-11eb-90da-d8ab7794a0a3.gif)

Reply:
Thanks. Done. 👍🏽